doc/README.rst

```diff
@@ -2,6 +2,9 @@
 Documentation Source
 ####################
 
```

Comment:
I would add the xmos only line here too

Reply:
FIxed in https://github.com/xmos/fwk_rtos/pull/214/files.